src/formTimingTracking.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Form timing tracking
+ * 
+ * This script analyses the interactions with <form> fields.
+ */
+
+ export default (subscribe, {formNameAttribute, fieldNameAttribute}) => {
+    let fieldTimings = {};
```

Comment:
I would suggest `fieldsTimings` instead because this object could have more than one field with it's "interaction timing".

Reply:
fine by me, changed